### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #283: Fix the command to launch test on `CONTTRIBUTING.md`
The current command to run tests in the `Dotnet` SDK does not work with a `Meilisearch` running locally or with docker. 
For this to work, the `Meilisearch` instance must be started with the specific docker-compose file in the directory such as:
```
docker-compose up
```

### PR DESCRIPTION
**Pull request by Smartmind12**
# Pull Request

## Related issue
Fixes #283 

## What does this PR do?
- Fix the command in Contributing.md to launch the tests.

## PR checklist
Please check if your PR fulfills the following requirements:
- [X] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [X] Have you read the contributing guidelines?
- [X] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by alallema** [comment]
Hello @Smartmind12,
Thank you very much for contributing to Meilisearch :heart:.
However, I am not available on the weekend, but I will be back on Monday :blush:.

*PS: This message was sent automatically!*

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,14 @@ You need to install the [.NET Core command-line interface (CLI) tools](https://d
 
 ### Tests <!-- omit in TOC -->
 
+Meilisearch instance must be started with the specific docker-compose file in the directory:
```
Comment (alallema) [inline CONTRIBUTING.md]:
```suggestion
To run the tests locally, the Meilisearch instance must be started with the specific docker-compose file in the:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,6 +58,14 @@ You need to install the [.NET Core command-line interface (CLI) tools](https://d
...
+
+```bash
+docker-compose up
+```
+
+You need to run the following command to launch tests in the Dotnet SDK:
+
 ```bash
 curl -L https://install.meilisearch.com | sh # download Meilisearch
 ./meilisearch --master-key=masterKey --no-analytics # run Meilisearch
```
Comment (alallema) [inline CONTRIBUTING.md]: This 2 lines are not needed if Meilisearch is launch with docker.

**Review by alallema — COMMENTED** [review]
Thanks, @Smartmind12 for this PR.
I just made a few suggestions.

**Comment by Smartmind12** [comment]
> Thanks, @Smartmind12 for this PR. I just made a few suggestions.

@alallema I have resolved the suggestions and updated the same ...Kindly review!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -58,9 +58,15 @@ You need to install the [.NET Core command-line interface (CLI) tools](https://d
 
 ### Tests <!-- omit in TOC -->
 
+To run the tests locally, the Meilisearch instance must be started with the specific docker-compose file in the:
```
Comment (alallema) [inline CONTRIBUTING.md]: Isn't a word missing?

Comment (Smartmind12) [reply CONTRIBUTING.md]: My bad ...I have updated the same!

**Comment by Smartmind12** [comment]
@alallema Kindly review now!

**Review by alallema — APPROVED** [review]
Thanks, @Smartmind12 for this PR!
LGTM!

**Comment by alallema** [comment]
bors merge

**Comment by alallema** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**